solo.json.erb

```diff
@@ -25,18 +25,8 @@ raise RuntimeError, 'environment variable NEXUS_DATA is required' if ENV['NEXUS_
   :java => {
     :jdk_version => ENV['JAVA_VERSION_MAJOR'],
     :java_home => ENV['JAVA_HOME'],
-    :install_flavor => 'oracle',
-    :oracle => {
-      :accept_oracle_download_terms => true
-    },
-    :jdk => {
-      :'8' => {
-        :x86_64 => {
-          :url => ENV['JAVA_URL'],
-          :checksum => ENV['JAVA_DOWNLOAD_SHA256_HASH']
-        }
-      }
-    }
+    :install_flavor => 'openjdk',
+    :jdk_version => '8'
```

Comment:
`jdk_version` defined above already... though I'm not sure where `JAVA_VERSION_MAJOR` is defined...

Reply:
Yeah, I couldn't find a way how to pin to a specific JDK release. If we need to do that we could either switch to using [OpenJDK](https://hub.docker.com/_/openjdk/?tab=description) as base image, but they are limited to Debian, Alpine or Oracle Linux plus not images aren't available for each JDK release. Alternatively we could use [Azul's OpenJDK](https://hub.docker.com/r/azul/zulu-openjdk) as base image as they have images for every (?) JDK release, but built on top of Ubuntu only.

Reply:
A popular choice these days is to use `adoptopenjdk/openjdk8:latest` as a base image. All other recent versions of Java are also made available by this community project [https://hub.docker.com/u/adoptopenjdk]().

Discussion on Reddit: [https://www.reddit.com/r/java/comments/9hd97k/openjdk_vs_adoptopenjdk/]()

Reply:
Sorry I didn't get back to you earlier, @geberl . Thanks you for pointing to AdoptOpenJDK. It's a great project, but I'm afraid we can't use it at this point in time. AdoptOpenJDK builds images for JDK 8 based on Alpine Linux and Ubuntu. Since we need Chef for system configuration, we can't use Alpine Linux at the moment. Internally, we run our tests against CentOS and Ubuntu, so we decided to stick to JDK installed by a Chef. Anyway, we'll keep our eye on how we can improve.